src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -47,7 +47,11 @@ export default function FieldTemplate(props) {
   let errorSpanId;
   let errorSpan;
   let errorClass;
-  if (hasErrors) {
+
+  if (
+    hasErrors &&
+    !(uiSchema && uiSchema['ui:options'] && uiSchema['ui:options'].hideErrors)
+  ) {
```

Comment:
This seems like a bad idea. Why would we want to hide the errors?

Reply:
For the markup hierarchy to work with the date widget within the review box (to display correctly), I had to manually hide the errors here and display them in the review box. This was only necessary for the date widget; the markup and errors displayed fine for the full name field. For how I displayed the errors within the Review Box, see ReviewBoxField.jsx, lines 437-464.

And I'm definitely open to an alternative implementation that would require this extra step.

Reply:
Commenting here for posterity: @nathanielbuck and I met and talked about it. What we did to fix the problem he was running into was to wrap the `dateOfBirth` in a `view:dateOfBirth` field and apply the `ReviewBoxField` to that parent object instead of directly on the `dateOfBirth`.